Source/Core/DolphinQt/GameList/GameListModel.cpp

```diff
@@ -89,6 +89,8 @@ QVariant GameListModel::data(const QModelIndex& index, int role) const
     if (role == Qt::DisplayRole || role == Qt::InitialSortOrderRole)
     {
       QString name = QString::fromStdString(game.GetName(m_title_database));
+
+      // Add disc number to title if not present.
```

Comment:
This comment is not entirely accurate. It doesn't add the disc number for disc 1.

Reply:
I suppose you could just move the comment inside the `if (disc_nr > 1)` part and it would be fine.

Reply:
I'll tweak the comment a bit. :P